project/iterative_sorting.py

```diff
@@ -5,15 +5,14 @@ def selection_sort( arr ):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
-
+        # (hint, can do in 3 loc)
```

Comment:
looks great Kyran, but I'm getting arrays sorted in descending order. can you make it go in ascending order?